week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/framework/dom-creators.ts

```diff
@@ -1,5 +1,5 @@
-function createElement(tagName = 'div', className = ''): HTMLDivElement | HTMLParagraphElement | HTMLElement {
-  const element = document.createElement(tagName);
+function createElement<T extends HTMLDivElement | HTMLParagraphElement | HTMLElement>(tagName = 'div', className = ''): T {
+  const element = <T>document.createElement(tagName);
```

Comment:
Dalej tego nie kupuje xD `asercja typów` to jest dalej tylko z drugiej strony (inny syntax niż `as`)